Its.Log.Monitoring/Telemetry.cs

```diff
@@ -0,0 +1,53 @@
+using System.Collections.Generic;
+using Its.Log.Instrumentation;
+
+namespace Its.Log.Monitoring
+{
+    public class Telemetry
+    {
+        private IDictionary<string, object> properties;
+
+        /// <summary>
+        ///     Initializes the <see cref="Instrumentation.Extensions.Telemetry" /> class.
+        /// </summary>
+        static Telemetry()
+        {
+            Formatter<Instrumentation.Extensions.Telemetry>.RegisterForAllMembers();
+        }
```

Comment:
There's a pattern in Its.Log where default formatters are registered in a single place and initialization is resettable, since these formatters can be overwritten by user code. Take a look at `Formatter.RegisterDefaults`. A similar or integrated approach might be good here.